spec/support/vcr.rb

```diff
@@ -0,0 +1,11 @@
+require "vcr"
+
+def vcr_record_mode
+  (ENV["VCR_RECORD"] || :none).to_sym
```

Comment:
I think it reads more clearly using `#fetch`. What do you think?

```suggestion
  ENV.fetch("VCR_RECORD", :none).to_sym
```

Reply:
I agree 👍